docker_cli/_image.py

```diff
@@ -0,0 +1,471 @@
+        stderr : io.TextIOBase or special value, optional
+            For a description of valid values, see :func:`subprocess.run`.
+        network : str, optional
+            The name of the network. Defaults to "host".
+        no-cache : bool, optional
+            A boolean designating whether or not the docker build should use
+            the cache.
+
+        Returns
+        -------
+        Image
+            The created image.
+
+        Raises
+        -------
+        DockerBuildError
+            If the docker build command fails.
+        ValueError
+            If both `dockerfile` and `dockerfile_string` are defined.
+        """
+        ...
+
+    @overload
+    @classmethod
+    def build(
+        cls: Type[Self],
+        tag: str,
+        *,
+        context: Union[str, os.PathLike[str]],
+        dockerfile_string: str,
+        stdout: Optional[io.TextIOBase],
+        stderr: Optional[io.TextIOBase],
+        network: str,
+        no_cache: bool
+    ) -> Self:
+        """
+        Builds a new image from a string in dockerfile syntax.
+
+        Parameters
+        ----------
+        tag : str
+            A name for the image.
+        context : os.PathLike, optional
+            The build context. Defaults to ".".
+        dockerfile_string : str
+            A Dockerfile-formatted string.
+        stdout : io.TextIOBase or special value, optional
+            For a description of valid values, see :func:`subprocess.run`.
+        stderr : io.TextIOBase or special value, optional
+            For a description of valid values, see :func:`subprocess.run`.
+        network : str, optional
+            The name of the network. Defaults to "host".
+        no-cache : bool, optional
+            A boolean designating whether or not the docker build should use
+            the cache.
+
+        Returns
+        -------
+        Image
+            The created Image.
+
+        Raises
+        -------
+        DockerBuildError
+            If the docker build command fails.
+        ValueError
+            If both `dockerfile` and `dockerfile_string` are defined.
+        """
+        ...
+
+    @classmethod
+    def build(
+        cls,
+        tag,
+        *,
+        context=".",
+        dockerfile=None,
+        dockerfile_string=None,
+        stdout=None,
+        stderr=None,
+        network="host",
+        no_cache=True
+    ):
+        if dockerfile is not None and dockerfile_string is not None:
+            raise ValueError(
+                "Both dockerfile and dockerfile_string passed as arguments."
+            )
+
+        # Build with dockerfile if dockerfile_string is None
+        dockerfile_build = dockerfile_string is None
+
+        context_str = os.fspath(".") if context is None else os.fspath(context)
+        cmd = [
+            "docker",
+            "build",
+            f"--network={network}",
+            context_str,
+            "-t",
+            tag
```

Comment:
Looks like you might've missed this suggested change (or perhaps it was accidentally reverted).

```suggestion
            f"-t={tag}"
```